setup.py

```diff
@@ -14,7 +14,7 @@
 
 setup(
     name='luminaire',
-    version='0.2.0.dev1',
+    version='0.2.1.dev1',
```

Comment:
Let's not bump version for a simple build system update.

Reply:
I like the new contributing doc pushed by @snazzyfox that one of the project owner can later bump the version when ready to release, and we avoid each contributor to bump the version.

Reply:
@shahsmit14 got it